axelrod/strategies/_strategies.py

```diff
@@ -105,6 +106,7 @@
     LimitedRetaliate,
     LimitedRetaliate2,
     LimitedRetaliate3,
+    EvolvedLookerUp,
```

Comment:
I think LookerUp isn't intended to be an ordinary strategy -- it would just default to TFT. It's still available as `axelrod.LookerUp`.